apps/experiments/models.py

```diff
@@ -1932,3 +1931,35 @@ def as_chatbot_chip(self) -> Chip:
             "chatbots:chatbot_session_view", args=[self.team.slug, self.experiment.public_id, self.external_id]
         )
         return Chip(label=self.external_id, url=url)
+
+class FilterSet(BaseTeamModel):
+    name = models.CharField(max_length=256)
+    user = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
+    table_type = models.CharField(max_length=50)
```

Comment:
Maybe use a `TextChoices` for the `table_type`. That way it's easier to see what we expect here and what it will be used for